docs/release/trg-0/trg-8-04.md

```diff
@@ -0,0 +1,167 @@
+---
+title: TRG 8.04 - KIT Structure
+---
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 18-Jan-2024  | Initial contribution                   |
+
+## Project Structure
+
+Following our project structure, the collection of our KITs documentation is developed in the `./docs-kits/kits` folder, where each KIT is a subfolder called by its name for organisation purposes. The `Data Chain KIT`, for example, is defined here: `./docs-kits/kits/Data Chain Kit`.
+
+Each folder/KIT's content is structured in at least four pages/subfolders:
+
+- Adoption View -> `page_adoption-view.md`
+- Software Operation View -> `page_software-operation-view.md`
+- Documentation -> `page_documentation.md`
+- Software Development View/
+  - Specification -> `page_software-development-view.md`
+  - OpenAPI definition/
```

Comment:
OpenAPI definitions are moving to [SwaggerHub](https://app.swaggerhub.com/search?owner=eclipse-tractusx-bot), so I would not mention any local structure here